charity.go

```diff
@@ -1,7 +1,7 @@
 package helix
 
 type CharityCampaignAmount struct {
-	Value         int64  `json:"value"`
+	Value         int64  `json:"amount"`
```

Comment:
Should the struct field name also be `Amount`?

Reply:
hey @nicklaw5 

To be honest this endpoint is still in BETA, up until this endpoint is of 'LIVE' status I will adjust accordingly,
as i'm not sure whether the json field will be value or amount thereafter 🤷

Reply:
https://github.com/twitchdev/issues/issues/642